src/components/CustomisedAppBar.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  AppBar,
+  Box,
+  Container,
+  Stack,
+  Toolbar,
+  Typography,
+} from '@mui/material'
+import CustomisedButton from './CustomisedButton'
+import {
+  fontColor,
+  fontFamily,
+  primaryColor,
+  secondaryColor,
+  backgroundColor,
+} from '../vars'
+import Logo from '../assets/Logo.svg'
+
+interface CustomisedAppBarProps {}
```

Comment:
Use interface for appbar from mui instead of empty interface

Reply:
fixed